exporter/otlp-metrics/opentelemetry-exporter-otlp-metrics.gemspec

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+lib = File.expand_path('lib', __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'opentelemetry/exporter/otlp/version'
+
+Gem::Specification.new do |spec|
+  spec.name        = 'opentelemetry-exporter-otlp-metrics'
+  spec.version     = OpenTelemetry::Exporter::OTLP::VERSION
+  spec.authors     = ['OpenTelemetry Authors']
+  spec.email       = ['cncf-opentelemetry-contributors@lists.cncf.io']
+
+  spec.summary     = 'OTLP metrics exporter for the OpenTelemetry framework'
+  spec.description = 'OTLP metrics exporter for the OpenTelemetry framework'
+  spec.homepage    = 'https://github.com/open-telemetry/opentelemetry-ruby'
+  spec.license     = 'Apache-2.0'
+
+  spec.files = ::Dir.glob('lib/**/*.rb') +
+               ::Dir.glob('*.md') +
+               ['LICENSE', '.yardopts']
+  spec.require_paths = ['lib']
+  spec.required_ruby_version = '>= 3.0'
+
+  spec.add_dependency 'googleapis-common-protos-types', '~> 1.3'
+  spec.add_dependency 'google-protobuf', '~> 3.14'
+  spec.add_dependency 'opentelemetry-api', '~> 1.1'
+  spec.add_dependency 'opentelemetry-common', '~> 0.20'
+  spec.add_dependency 'opentelemetry-metrics-api'
+  spec.add_dependency 'opentelemetry-metrics-sdk'
```

Comment:
Probably should remove these dependencies? 

Since there is no metrics-related gem in rubygems, user who install this and use it will fail.

User who want to use it should create the metrics gem by themselves, and require metrics gem first, and then this metrics exporter.